modules/virt-creating-offline-vm-snapshot-cli.adoc

```diff
@@ -1,11 +1,22 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-managing-offline-vm-snapshots.adoc
+// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
 
-[id="virt-creating-offline-vm-snapshot-cli_{context}"]
-= Creating an offline virtual machine snapshot in the CLI
+[id="virt-creating-vm-snapshot-cli_{context}"]
+= Creating an virtual machine snapshot in the CLI
 
-You can create a virtual machine (VM) snapshot for an offline VM by creating a `VirtualMachineSnapshot` object.
+You can create a virtual machine (VM) snapshot for a VM by creating a `VirtualMachineSnapshot` object.
+
+You can create a virtual machine (VM) snapshot for an online VM by creating a `XX` object. Kubevirt will coordinate with the QEMU guest agent to create a snapshot of the online VM.
```

Comment:
It is the same object regardless of the state of the VM (running or stopped)

Reply:
Sorry.... I made this change to another module and forgot to copy it. It should now read:

You can create a virtual machine (VM) snapshot for an online or offline VM by creating a `VirtualMachineSnapshot` object. Kubevirt will coordinate with the QEMU guest agent to create a snapshot of the online VM.